test/functional/sandbox/html/bogusCommand.html

```diff
@@ -20,8 +20,7 @@
         configId: "9999999",
         orgId: "53A16ACB5CC1D3760A495C99@AdobeOrg",
         debugEnabled: true,
-        prehidingStyle: "section:nth-of-type(1) { opacity: 0 !important }",
-        idSyncContainerId: 81
+        prehidingStyle: "section:nth-of-type(1) { opacity: 0 !important }"
```

Comment:
We have the configId 9999999 defined to get the idSyncContainerId 81 back?

Reply:
Yes that should be configured on the server.  But I don't think Alloy cares what the idSyncContainerId is at all.